src/coreclr/src/gc/gc.cpp

```diff
@@ -38347,3 +38374,22 @@ void PopulateDacVars(GcDacVars *gcDacVars)
     UNREFERENCED_PARAMETER(gcDacVars);
 #endif // DACCESS_COMPILE
 }
+
+gc_oh_num gen_to_oh(int gen)
```

Comment:
I would move this toward the beginning of the file as the later file is for GCHeap and other implementations